python/lsst/ctrl/execute/allocator.py

```diff
@@ -48,11 +49,18 @@ class Allocator:
         the name of the platform to execute on
     opts : `Config`
         Config object containing options
-    condorInfoFileName : `str`
+    condorInfoFileName : `str | lsst.resources.ResourcePath`
```

Comment:
```suggestion
    condorInfoFileName : `str` | `lsst.resources.ResourcePath`
```
You can instead use `lsst.resources.ResourcePathExpression`